app/src/components/Modals/activity-modal/activity-form-modal.tsx

```diff
@@ -168,13 +171,18 @@ const AddActivityModal: FC<AddActivityModalProps> = ({
 
     // so the issue here is that we need to have one inventoryValue for
     const requestData = {
-      activityData: {
-        co2_amount: gasValues[1].factor,
-        ch4_amount: gasValues[0].factor,
-        n2o_amount: gasValues[2].factor,
-        ...values,
+      activityData: methodology?.id.includes("direct-measure")
+        ? {
+            co2_amount: gasValues[1].factor,
+            ch4_amount: gasValues[0].factor,
+            n2o_amount: gasValues[2].factor,
+            ...values,
+          }
+        : { ...values },
+      metadata: {
+        emissionFactorType: activity.emissionFactorType,
+        totalFuelConsumption: activity.totalFuelConsumption,
```

Comment:
`totalFuelConsumption` should be in the activity data, not metadata. Emissions factor type makes sense though, since it's not directly used in the calculation (only used to restore form state when editing, since choosing an emissions factor means its values will be loaded into the 3 emissions factor form inputs.
I think these field names should all be loaded from the manual input hierarchy JSON file?
Because not every activity has totalFuelConsumption, only some methodologies use this.

Reply:
This falls under this ticket: https://openearth.atlassian.net/browse/ON-2331?atlOrigin=eyJpIjoiMjdhMDdmNWNlMTRlNDFlZjkwNDM3ZGJmMjQzYzhmMjkiLCJwIjoiaiJ9